lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/initial-journey-selection.yaml

```diff
@@ -54,6 +54,9 @@ CHECK_EXISTING_IDENTITY:
     enhanced-verification-f2f-fail:
       targetJourney: NEW_P2_IDENTITY
       targetState: ENHANCED_VERIFICATION_F2F_FAIL
+    repeat-fraud-check:
+      targetJourney: REPEAT_FRAUD_CHECK
+      targetState: CONFIRM_NAME_DOB
```

Comment:
I think this can just be `START`, and the entry routing in the journey definition will take care of the rest

Reply:
Yea, thanks Roger. Noticed this earlier and fixed it already.